### ISSUE TITLE
train using multiple datasets

### ISSUE THREAD

**Issue by jeff-regier** [YosefLab/scVI-dev#82]
PyTorch's `ConcatDataset` utility lets us combine two dataset objects into one, so we can train using both together. e.g., one of the datasets has annotations (labels) and one does not.

http://pytorch.org/docs/master/data.html#torch.utils.data.ConcatDataset
